rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,57 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+# commit value is dynamic, see build script:
+# https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
```

Comment:
Aren't we `sed` `global commit`, not just `commit`? O_o @Sevlin @EvgeniyPatlan how that works?

Reply:
Version and commit values are being `sed`'ed during build :\

Reply:
Where?

Reply:
In procedure `prepare_specs()`: https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm#L59

Reply:
So that line literally `sed`s `global commit`, not just `commit`. Hence my question

Reply:
bah, my eyes, get it now